tutorials/lesson3-anatomy/coin.md

```diff
@@ -279,15 +290,15 @@ It is often convenient to define a few helper operators.
 
 We start with the definition of `state` that represents the entire
 state as a record. This is what we do in the above code with the definition of
-`state`. Notice that the definition of `state` is not `pure` anymore. It is
-referring to the state variables and thus cannot be pure.
+`state`. Notice that the definition of `state` is prefixed with `val`, not `pure val`.
+Since `state` accesses state variables is it impure.
```

Comment:
```suggestion
`state`. Notice that the definition of `state` is prefixed with `val`, not `pure val`:
since `state` accesses state variables, is it impure.
```